dev/integration_tests/android_embedding_v2_smoke_test/pubspec.yaml

```diff
@@ -19,20 +19,16 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  # This plugin is using Android Embedding 1
-  battery: 2.0.3
```

Comment:
Should this instead be replaced with https://pub.dev/packages/battery_plus

Also was this change done by the script?

Reply:
Sorry this change was also not done by the script (I don't know how I forgot both of these). This plugin was also discontinued, so I just tried removing it. It doesn't look like it is actually used. 

I think the purpose of including it was just to make sure a v2 embedding app could use a v1 embedding plugin, but that doesn't matter anymore because I've removed the logic it would be testing.